beancount/parser/lexer_test.py

```diff
@@ -429,13 +427,11 @@ def test_null_true_false(self, tokens, errors):
         '''
         TRUE FALSE NULL
         '''
-        # Note that this test contains an _actual_ newline, not an escape one as
-        # in the previous test. This should allow us to parse multiline strings.
         self.assertEqual([
             ('EOL', 2, '\n', None),
             ('BOOL', 2, 'TRUE', None),
             ('BOOL', 2, 'FALSE', None),
-            ('NULL', 2, 'NULL', None),
+            ('NONE', 2, 'NULL', None),
```

Comment:
Why this change? I'm calling them "NULL" in Beancount.

Reply:
The token is called NONE in the grammar definition https://github.com/beancount/beancount/blob/master/beancount/parser/grammar.y#L211 , its name is translated into "NULL" in the function that this commits drops https://github.com/beancount/beancount/blob/master/beancount/parser/grammar.y#L941 I didn't fee like this is important enough to rename the token or special case it in the `token_to_string()` function. Either can be done if you think the name NULL is more appropriate.

Reply:
Fair enough.